totalRP3/Modules/Toolbar/Toolbar.lua

```diff
@@ -92,6 +94,7 @@ local function onStart()
 		assert(LDBButton, "Could not find a registered LDB object for id " .. buttonStructure.id)
 
 		LDBButton.icon = Utils.getIconTexture(buttonStructure.icon);
+		LDBButton.text = buttonStructure.text; -- need to manually update each attribute here in order to correctly propagate object changes
```

Comment:
```suggestion
		LDBButton.text = buttonStructure.text;
```
Superfluous comment; it should be no surprise that a function whose sole purpose is to update LDB attributes has to indeed update attributes.